lib/bin.ts

```diff
@@ -0,0 +1,86 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { resolve } from 'path';
+import { parseArgs } from 'util';
+import { runTests } from './runTest';
+
+// TypeScript doesn’t allow imports outside the root dir.
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const pkg = require('../package.json');
+
+const options = {
+	'vscode-executable-path': { type: 'string' },
+	'vscode-version': { type: 'string' },
+	platform: { type: 'string' },
+	'reuse-machine-install': { type: 'boolean' },
+	'extension-development-path': { type: 'string', multiple: true },
```

Comment:
Did you intentionally miss this one in help? Also, is it intentionally made "multiple"?

Reply:
Good catch! I accidentally missed that one.

It’s `multiple`, because `runTests()` accepts it as an array.